### PR TITLE
Fix global brightness

### PR DESCRIPTION
**Pull request by wuan**
This PR fixes setting initial values and persisting of global brightness value when switching shows.

### CODE REVIEW

File: server/lightshows/christmas.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def init_parameters(self):
                            "chunk_blendover": 5,
                            "whole_blendover": 5}
 
-    def set_parameter(self, param_name: str, value):
+    def set_parameter(self, param_name: str, value, **kwargs):
```
Comment (sleiner) [inline server/lightshows/christmas.py:20]: How does this help syncing the brightness?

File: server/mqttcontrol.py [hunk 2]
```diff
@@ -178,8 +178,8 @@ def run(self) -> None:
         driver = get_driver(self.conf.Strip.driver)
         self.strip = driver(num_leds=self.conf.Strip.num_leds,
                             max_clock_speed_hz=self.conf.Strip.max_clock_speed_hz,
-                            max_global_brightness=self.conf.Strip.max_brightness_percent)
-        self.strip.set_global_brightness_percent(self.conf.Strip.initial_brightness_percent)
+                            max_global_brightness=self.conf.Strip.max_brightness_percent / 100.0)
+        self.strip.set_global_brightness_percent(self.conf.Strip.initial_brightness_percent / 100.0)
```
Comment (sleiner) [inline server/mqttcontrol.py]: Please use `set_global_brightness()` instead and remove `set_global_brightness_percent()` as this is the only point of usage anyway.

**Review by sleiner — CHANGES_REQUESTED** [review]
Thank you very much! 👍 
There are still two small things that I would like you to address. Then i will happily merge the very first pull request 🥇

File: server/lightshows/christmas.py (hunk 1 above)
Comment (wuan) [reply server/lightshows/christmas.py:20]: Not really, but I had to add this in the beginning to get it running. I will have a closer look this evening.

File: server/mqttcontrol.py (hunk 2 above)
Comment (wuan) [reply server/mqttcontrol.py]: Will do that.

**Comment by wuan** [comment]
And thank you for the idea and the good work!

File: server/lightshows/christmas.py (hunk 1 above)
Comment (wuan) [reply server/lightshows/christmas.py:20]: `Christmas` overrides the method `set_parameters` from `Lightshow` which is called from `apply_parameter`.

Without the fix running the christmas show leads to the following error:
```
...
  File ".../102shows/server/lightshows/templates/base.py", line 70, in __init__
    self.apply_parameter_set(parameters)
  File ".../102shows/server/lightshows/templates/base.py", line 193, in apply_parameter_set
    self.set_parameter(param_name, value=parameters[param_name], send_mqtt_update=False)
TypeError: set_parameter() got an unexpected keyword argument 'send_mqtt_update'
```

As this call seemed not to be how it is intended adding `**kwargs` was the simplest way to fix the Christmas show.

Comment (sleiner) [reply server/lightshows/christmas.py:20]: I see. I added a call to the overridden method, for potential parameters that the overriding method does not cover.
I will fix this properly (using `Lightshow.p`) next week

**Comment by wuan** [comment]
Is there anything which is missing?

**Comment by wuan** [comment]
There is still the question if you are missing something? Any response would be helpful.

**Comment by sleiner** [comment]
I'm sorry so sorry for having you kept waiting for so long - the last months have been very busy for me. Unfortunately when i tried to test the code from this branch I realized that my test system broke in the mean time. I'm currently repairing it and will test it in the next few days. If that succeeds, we can _finally_ merge the pull request.

**Comment by wuan** [comment]
Any news from your test system yet?

**Review by sleiner — APPROVED** [review]

**Comment by sleiner** [comment]
I could never really get it to work again; after trying a few hours today, it seems like my strips have not taken the heat well :/ But since this works fine on your end, let's merge this (at last!)

**Pull request merged**